doc-content/enterprise-only/employee-rostering/er-contracts-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='er-contracts-proc']
+= Entering the list of contracts
+
+You must enter the list of all types of contracts that the business uses for employees.
+
+A contract determines the maximum time that the employee can work in a day, calendar week, calendar month, or calendar year.
+
+You can set any of the limitations or none at all is a contract. For example, a part-time employee might not be allowed to work more than 20 hours per week, while a full-time employee might be limited to 10 hours in a day and 1800 hours during a year. Another contract might include no limitations on worked hours.
+
+You must enter all work time limits for contracts in minutes.
+
+.Procedure
+
+. To enter or change the list of contracts, click the *Contracts* tab.
++
+You can see the numbers of currently visible contracts in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other contracts in the list.
++
+You can enter any part of an contract name in the *Search* box to search for contracts.
++
+. Complete the following steps to add a new contract:
+.. Click *Add Contract*.
+.. Enter the name of the contract into the text field under *Name*.
```

Comment:
I'd say:
 Enter the contract name in the *Name* box. In any case, change into to to.